examples/nas/multi-trial/mnasnet/base_mnasnet.py

```diff
@@ -180,7 +180,7 @@ def __init__(self, alpha, depths, convops, kernel_sizes, num_layers,
             nn.ReLU(inplace=True),
         ]
         self.layers = nn.Sequential(*layers)
-        self.classifier = nn.Sequential(nn.Dropout(p=dropout, inplace=True),
```

Comment:
why adding `inplace=True` induces problem?

Reply:
Error info says 'one of the variables needed for gradient computation has been modified by an inplace operation.'
The new version of PyTorch does not support inplace operation, so that the back propagation is interrupted by the `inplace` operation in dropout layer. Here is a similar issue in pytorch: https://discuss.pytorch.org/t/help-runtimeerror-one-of-the-variables-needed-for-gradient-computation-has-been-modified-by-an-inplace-operation/107030/2.

Reply:
so this is an issue of pytorch not nni, right?